src/ansys/acp/core/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 import importlib.metadata
 
-from . import example_helpers, material_property_sets
+from . import example_helpers, material_property_sets, mechanical_integration_helpers
```

Comment:
FYI: example_helpers no longer exist. Just remove it from this list.

Reply:
Yup, I hadn't merged the latest changes from `main` yet.